CRM/Civicase/Service/CaseCategoryMenu.php

```diff
@@ -68,12 +68,12 @@ protected function createCaseCategorySubmenus($caseTypeCategoryName, array $perm
   }
 
   /**
-   * Update the submenus of the given category.
+   * Reset the submenus of the given category.
    *
    * @param string $caseTypeCategoryName
    *   Case type category name.
    */
-  public function updateCaseCategorySubmenus(string $caseTypeCategoryName) {
+  public function resetCaseCategorySubmenus(string $caseTypeCategoryName) {
```

Comment:
Sorry to be a bother but can you update this to `resetCaseCategorySubmenusUrl` like I suggested or something better to reflect that it is the URL that is being reset? Thanks

Reply:
Changed: https://github.com/compucorp/uk.co.compucorp.civicase/pull/756/commits/ae8a1f21254cc92a3a2737485c645d3dfce62689 I did not notice on your first comment the intention of making reference to the Url, which is true of course.